### PR TITLE
style edits to performance metrics docs

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

File: src/platforms/common/performance/instrumentation/performance-metrics.mdx [hunk 1]
```diff
@@ -36,23 +36,24 @@ Sentry's SDKs support sending performance metrics data to Sentry. These are nume
...
 - value (any numeric type - `float`, `integer`, etc.)
-- unit (`string`, can be empty or the string `'none'`). 
+- unit (`string`, can be empty or the string `'none'`).
 
 Sentry supports adding arbitrary custom units, but we recommend using one of the [supported units listed below](./#supported-performance-metric-units).
 
 <PlatformContent includePath="performance/custom-performance-metrics" />
 
-<Alert level="warning" title="Note">
+<Note>
```
Comment (imatwawana) [inline src/platforms/common/performance/instrumentation/performance-metrics.mdx:48]: @lobsterkatie - Mostly just need confirmation that it's okay for me to change this to a regular note. It doesn't seem to warrant red background.

File: src/includes/performance/custom-performance-metrics/javascript.mdx [hunk 2]
```diff
@@ -1,14 +1,16 @@
-Supported in Sentry's JavaScript SDK version `7.0.0` and above.
+Adding custom metrics is supported in Sentry's JavaScript SDK version `7.0.0` and above.
```
Comment (imatwawana) [inline src/includes/performance/custom-performance-metrics/javascript.mdx:1]: @lobsterkatie - Please confirm I haven't changed the meaning here.

File: src/platforms/common/performance/instrumentation/performance-metrics.mdx (hunk 1 above)
Comment (lobsterkatie) [reply src/platforms/common/performance/instrumentation/performance-metrics.mdx:48]: Oh, yeah, whoops. I was figuring "warning" was probably going to be yellow. Red does seem excessive.

File: src/includes/performance/custom-performance-metrics/javascript.mdx [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 
 ```javascript
-const transaction = Sentry.getCurrentHub().getScope().getTransaction();
+const transaction = Sentry.getCurrentHub()
+  .getScope()
+  .getTransaction();
 
 // Record amount of memory used
-transaction.setMeasurement('memoryUsed', 123, 'byte');
+transaction.setMeasurement("memoryUsed", 123, "byte");
```
Comment (lobsterkatie) [inline src/includes/performance/custom-performance-metrics/javascript.mdx:9]: I'm curious if this is auto-formatting or a conscious choice. (In the JS SDK, our standard is single quotes. Makes no difference to the computer, so neither one is "right," but it means that any code that we copy in from our end is going to have single quotes.)

**Review by lobsterkatie — APPROVED** [review]

File: src/includes/performance/custom-performance-metrics/javascript.mdx (hunk 2 above)
Comment (imatwawana) [reply src/includes/performance/custom-performance-metrics/javascript.mdx:1]: Nah, I think let's leave it above the snippet so they don't start copying and pasting without knowing if they can use it.

File: src/includes/performance/custom-performance-metrics/javascript.mdx (hunk 3 above)
Comment (imatwawana) [reply src/includes/performance/custom-performance-metrics/javascript.mdx:9]: Definitely auto-formatting. I don't love the idea of it being inconsistent (though I'm sure I've already caused this with my linter), and I think y'all should be the authority on which types of quotes we use, so I'm gonna kill these edits.

Comment (imatwawana) [inline src/includes/performance/custom-performance-metrics/javascript.mdx:9]:
```suggestion
```

File: src/includes/performance/custom-performance-metrics/javascript.mdx [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
+Adding custom metrics is supported in Sentry's JavaScript SDK version `7.0.0` and above.
 
 ```javascript
-const transaction = Sentry.getCurrentHub().getScope().getTransaction();
+const transaction = Sentry.getCurrentHub()
+  .getScope()
+  .getTransaction();
 
 // Record amount of memory used
-transaction.setMeasurement('memoryUsed', 123, 'byte');
```
Comment (imatwawana) [inline src/includes/performance/custom-performance-metrics/javascript.mdx:9]:
```suggestion
transaction.setMeasurement('memoryUsed', 123, 'byte');
```

File: src/includes/performance/custom-performance-metrics/javascript.mdx [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
+const transaction = Sentry.getCurrentHub()
+  .getScope()
+  .getTransaction();
 
 // Record amount of memory used
 transaction.setMeasurement('memoryUsed', 123, 'byte');
-
 // Record time when Footer component renders on page
-transaction.setMeasurement('ui.footerComponent.render', 1.3, 'second');
+transaction.setMeasurement("ui.footerComponent.render", 1.3, "second");
```
Comment (imatwawana) [inline src/includes/performance/custom-performance-metrics/javascript.mdx:11]:
```suggestion
transaction.setMeasurement('ui.footerComponent.render', 1.3, 'second');
```

File: src/includes/performance/custom-performance-metrics/javascript.mdx [hunk 6]
```diff
@@ -1,14 +1,15 @@
...
 
 // Record amount of times localStorage was read
-transaction.setMeasurement('localStorageRead', 4);
+transaction.setMeasurement("localStorageRead", 4);
```
Comment (imatwawana) [inline src/includes/performance/custom-performance-metrics/javascript.mdx:14]:
```suggestion
transaction.setMeasurement('localStorageRead', 4);
```

File: src/includes/performance/custom-performance-metrics/javascript.mdx [hunk 7]
```diff
@@ -1,11 +1,10 @@
-Supported in Sentry's JavaScript SDK version `7.0.0` and above.
+Adding custom metrics is supported in Sentry's JavaScript SDK version `7.0.0` and above.
 
 ```javascript
 const transaction = Sentry.getCurrentHub().getScope().getTransaction();
 
 // Record amount of memory used
 transaction.setMeasurement('memoryUsed', 123, 'byte');
```
Comment (imatwawana) [inline src/includes/performance/custom-performance-metrics/javascript.mdx:7]:
```suggestion
transaction.setMeasurement('memoryUsed', 123, 'byte');

```

**Pull request merged**